.travis.yml

```diff
@@ -1,12 +1,37 @@
 ---
 language: ruby
-rvm:
-  - 2.1.0
-script:
-  - bundle install
-  # Coding standards.
-  - touch /tmp/lint.txt
-  - for manifest in $(find manifests -name "*.pp"); do echo "Checking ${manifest}"; bundle exec puppet-lint --no-80chars-check --fail-on-warnings --no-autoloader_layout-check --error-level=all "${manifest}" | tee -a /tmp/lint.txt; done
-  - if [ $(cat /tmp/lint.txt | wc -l) -gt 0 ]; then exit 1; fi
-  # Syntax
-  - bundle exec rake syntax || exit 1
+bundler_args: --without development
+before_install: rm Gemfile.lock || true
+matrix:
+  include:
+  - rvm: 1.9.3
+    env: PUPPET_GEM_VERSION="~> 3.2.0"
```

Comment:
Is there a reason you're testing 3.2? It's a pretty old, there's not much benefit testing it because it's missing out a lot of essentials such as `contain`

Generally we test 3.8, 3.8 Future parser, latest 4.0?